ucte/ucte-converter/src/test/java/com/powsybl/ucte/converter/UcteImporterTest.java

```diff
@@ -176,5 +179,16 @@ public void testVoltageRegulatingXnode() {
         assertEquals(1.0, dl.getGeneration().getReactiveLimits().getMaxQ(dl.getGeneration().getTargetP()), 0.01);
         assertEquals(-1.0, dl.getGeneration().getReactiveLimits().getMinQ(dl.getGeneration().getTargetP()), 0.01);
     }
+
+    @Test
+    public void substationNameInvariance() {
+        ResourceDataSource dataSource = new ResourceDataSource("VLsCreationIssue", new ResourceSet("/", "VLsCreationIssue.uct"));
+
+        IntStream.iterate(0, i -> i + 1).limit(100).forEach(i -> {
```

Comment:
Use `IntStream.range(0, 100)` instead of `IntStream.iterate(0, i -> i + 1).limit(100)`

Reply:
Done